app/mailers/participant_mailer.rb

```diff
@@ -3,21 +3,20 @@
 class ParticipantMailer < ApplicationMailer
   PARTICIPANT_REMOVED_BY_STI = "ab8fb8b1-9f44-4d27-8e80-01d5d70d22f6"
 
-  PARTICIPANT_ADDED_TEMPLATES = {
-    ect: "50ee41e5-06b9-41cf-9afd-d5bc4db356c4",
-    mentor: "e0198213-c09d-41aa-8197-b167e495e49d",
-    sit_mentor: "7e7d3fdb-41f5-4e04-a4ae-acf92e8fefe6",
+  PARTICIPANT_TEMPLATES = {
+    ect_cip: "i3bfad27e-e7af-4b53-94cb-af36362f43d8",
```

Comment:
This `i` doesn't look right?

Reply:
How did you catch that? :D

Reply:
I was checking all the template IDs matched up with the names